homeassistant/components/voip/const.py

```diff
@@ -12,4 +12,7 @@
     "sleep_ratio": 0.99,
 }
 
+DEFAULT_SIP_USER = "HA"
```

Comment:
Could this impact how systems respond? Why not make the user optional if it's not set.